api/src/main/java/org/openmrs/module/registrationcore/api/db/RegistrationCoreDAO.java

```diff
@@ -22,6 +23,8 @@
  */
 public interface RegistrationCoreDAO {
 
+    DbSessionFactory getSessionFactory();
```

Comment:
You don't need this method on the DAO

Reply:
Could you explain a bit more? when I remove it I get a lot of errors because the hibernate session factory which implements this is used in quite a lot of places...

Reply:
It shouldn't be on the interface but rather the implementing class as a private method